src/pages/ExpensePage.jsx

```diff
@@ -1,5 +1,37 @@
-function Expense() {
-  return <h2>Expense</h2>;
-}
+import Table from "../components/Table";
+
+const rows = [
+  { id: 1, name: "ExpenseGroupName", description: "Description" },
+  { id: 2, name: "ExpenseGroupName22222", description: "Description222" },
+];
+const columns = [
```

Comment:
We could extract somewhere table configuration so that our component will be more readable and just import it from another place. You can create maybe a common folder where you can have different reusable things such as table config for example.